src/tvdb/models.py

```diff
@@ -0,0 +1,66 @@
+from pydantic import BaseModel
+
+
+class _RemoteID(BaseModel):
+    id: str
+    type: int
+    sourceName: str  # noqa: N815
+
+
+class _Translation(BaseModel):
+    ara: str | None = None
+    ces: str | None = None
+    deu: str | None = None
+
+
+class SearchData(BaseModel):
+    """Model for the data objects returned by the TVDB search endpoint."""
+
+    aliases: list[str]
+    companies: list[str] | None = None
+    companyType: str | None = None  # noqa: N815
```

Comment:
You can noqa this for the whole file with `# ruff: noqa: N815` on the first line